src/code/qualification/plotQualificationDDIRatio.m

```diff
@@ -181,7 +199,11 @@
         for k=1:length(PKParameter)
             % Get the PK parameters requested in PKParameter
             if strcmpi(PKParameter{k}, 'AUC')
-                PKpredField{k}= 'AUC_last';
+                if isinf(DDIRatios(j).SimulationControl.EndTime) && isinf(DDIRatios(j).SimulationDDI.EndTime)
+                    PKpredField{k}= 'AUC_inf';
+                else
+                    PKpredField{k}= 'AUC_last';
+                end

```

Comment:
@sfrechen Is this how it is expected to work?

Reply:
@sfrechen Please comment on this

Reply:
After discussing with @sfrechen : this is not correct.
I will merge the PR now to move forward and create a separate issue for this